platforms/android/library/build.gradle

```diff
@@ -51,6 +51,11 @@ android {
 
     buildFeatures {
         buildConfig = true
+        compose true // Enable classes to be marked as stable
+    }
+
+    composeOptions {
+        kotlinCompilerExtensionVersion '1.5.1'
```

Comment:
NIT: we are at 1.5.3 now.

Reply:
Thanks, I used this version to align with the other modules in the project but will update separately :+1: